cli/kontena-cli.gemspec

```diff
@@ -22,15 +21,16 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.7"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_runtime_dependency "excon", "~> 0.49.0"
+  spec.add_development_dependency "rake-compiler", "~> 1.0"
+  spec.add_runtime_dependency "excon", ">= 0.49.0", "< 0.50"
   spec.add_runtime_dependency "tty-prompt", "~> 0.10"
-  spec.add_runtime_dependency "clamp", "~> 1.1.0"
-  spec.add_runtime_dependency "ruby_dig", "~> 0.0.2"
-  spec.add_runtime_dependency "launchy", "~> 2.4.3"
-  spec.add_runtime_dependency "hash_validator", "~> 0.7.0"
-  spec.add_runtime_dependency "retriable", "~> 2.1.0"
+  spec.add_runtime_dependency "clamp", ">= 1.1", "< 1.2"
+  spec.add_runtime_dependency "ruby_dig", "< 0.1"
+  spec.add_runtime_dependency "launchy", ">= 2.4.3", "< 2.5"
+  spec.add_runtime_dependency "hash_validator", ">= 0.7", "< 0.8"
+  spec.add_runtime_dependency "retriable", ">= 2.1", "< 2.2"
```

Comment:
~> 2.3 means "equal to 2.3 or greater than 2.3, but less than 3.0", while ~> 2.3.0 would mean "equal to 2.3.0 or greater than 2.3.0, but less than 2.4.0

So while these `~> 2.3.0` are working, and getting 2.3.* like intended, gem build warns about them.